roles/4-server-options/tasks/main.yml

```diff
@@ -32,28 +32,12 @@
   when: usb_lib_install | bool
   #tags: usb-lib
 
-# NETWORK moved to the very end, after Stage 9 (9-LOCAL-ADDONS)
-# It can also be run manually using: cd /opt/iiab/iiab; ./iiab-network
-#
-#- name: NETWORK
-#  include_role:
-#    name: network
-#  # has no "when: XXXXX_install" flag
-#  tags: base, network
-
-# MANDATORY SO PERHAPS THIS BELONGS IN 3-BASE-SERVER ?
```

Comment:
I'll add a comment explaining this is in Stage 4-SERVER-OPTIONS because variable iiab_home_url change, and so this may need to be re-run in the field.